components/director/internal/domain/scenarioassignment/fixtures_test.go

```diff
@@ -4,11 +4,15 @@ import (
 	"context"
 	"errors"
 
+	"github.com/DATA-DOG/go-sqlmock"
+
 	"github.com/kyma-incubator/compass/components/director/internal/domain/scenarioassignment"
 	"github.com/kyma-incubator/compass/components/director/internal/domain/tenant"
 	"github.com/kyma-incubator/compass/components/director/internal/model"
 )
 
+const DefaultTenant = "tenant"
```

Comment:
Please make this TenantId and use a value more like UUID to underline this fact.